### DISCUSSION TITLE
Remember a selected filter or pagination status when navigating & modifying resources

### DISCUSSION THREAD

**Discussion by eAvio** [laravel/nova-issues#4673] [Ideas & Feature Requests]
I know this is duplicate of https://github.com/laravel/nova-issues/issues/1925

But that treat has been closed and marked as implemented. Although we can't seems to find this true. At least in [4.12.14](https://nova.laravel.com/releases/4.12.14). After saving/updating we get to detail view, but from here we can not get back to index (at least not without clicking two times back button in the browser.)

It would be more convenient to get back to index if you came from index and to detail view if you came from detail view. At lease in our case.  All the filters/search and pagination should be retained.

If I'm mistaken and this is actually implemented, could you please let us know how it works.

I marked this as a bug as we get daily complains from client about this behaviour - imagine they have to edit 100 of items from one resource and always loosing pagination and filters.

**Comment by RhysLees** [comment]
You can overwrite the behaviour of the redirects not sure how you would go about conditionally redirecting based on the current path though:

https://nova.laravel.com/docs/4.0/resources/#redirection

**Comment by stephaneisenbach** [comment]
Not implemented and not planned. See #6344

Does anyone have a workaround?

**Comment by scramatte** [comment]
It's absolutely necessary to be able to remember filters and pagination.

**Comment by stephaneisenbach** [comment]
If it is generally recognised as unexpected behavior (see https://github.com/laravel/nova-issues/issues/1925#issuecomment-530877390), than why not implement it as an optional feature to enable by configuration? There are many users who really miss this function.

**Reply by scramatte** [reply]
Imagine you have hundreds of items in a resource and you make some search and apply filters  for example with date range to reduce amount of data. Each time you update or create you loose all the previously created context.  It's very annoying 
The problem can be extended  to a simple search or sort by column.  All filters you apply on the index view should be restored when you come back from  edit or create.
